client/src/components/modal-forms/PSIForm.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import { Button } from '../generic';
+import { Box } from '@material-ui/core';
+import { RenderTextField, RenderSelectField } from '../fields';
+import { Field, Formik, Form as FormikForm } from 'formik';
+import { CreatePSISchema } from '../../constants';
+import { createPSI, updatePSI } from '../../services';
+
+const PSI_INITIAL_VALUES = {
+  instituteName: '',
+  streetAddress: '',
+  city: '',
+  postalCode: '',
+  healthAuthority: '',
+};
+
+const HEALTH_AUTHORITIES = [
+  { value: 'Interior', label: 'Interior Health' },
+  { value: 'Fraser', label: 'Fraser Health' },
+  { value: 'Vancouver Coastal', label: 'Vancouver Coastal Health' },
+  { value: 'Vancouver Island', label: 'Vancouver Island Health' },
+  { value: 'Northern', label: 'Northern Health' },
+];
```

Comment:
Did a quick repo search: an identical array is being used in several other spots on the client side- can we make it a constant?